2024-08-21 11:36:15,308 - __main__ - INFO - Starting sCT generation process...
2024-08-21 11:39:50,941 - __main__ - ERROR - Error during main process: [Errno 2] No such file or directory: '/data/galaponav/output/newHN_MR2/weights/DCNN_weights_best.pth'
